reference/tools/data-migration/dm-portal.md

```diff
@@ -0,0 +1,193 @@
+
+- If you run DM Portal locally, visit `127.0.0.1:8280` in your browser.
+- If you run DM Portal on a server, you need to configure a proxy on the server.
+
+### Deploy using DM Ansible
+
+To deploy DM Portal using DM Ansible, refer to [Deploy Data Migration Using DM-Ansible](/how-to/deploy/data-migration-with-ansible.md) for details.
+
+## Usage
+
+### Create rules
+
+This feature is used to create a `task.yaml` file. You need to select the replication mode, configure the upstream and downstream instances, configure table routing, and configure the binlog filter.
+
+#### Operation steps
+
+Access the dm-portal page, and click **Create New Rule**.
+
+### Configure the basic information
+
+This feature is used to fill in the task name and select a task type.
+
+#### Prerequisites
+
+**Create New Sync Rule** is already selected.
+
+#### Operation steps
+
+1. Fill in the task name.
+2. Choose a task type.
+
+![DM Portal BasicConfig](/media/dm-portal-basicconfig.png)
+
+### Configure the instance information
+
+This feature is used to configure the upstream and downstream instance information, including Host, Port, Username, and Password.
+
+#### Prerequisites
+
+**Task Name** and **Task Type** are already filled in.
+
+> **Note:**
+>
+> If you choose **Incremental** or **All** in Task Type, you need to configure binlog-file and binlog-pos when configuring the upstream instance information.
+
+#### Operation steps
+
+1. Fill in the upstream instance information.
+2. Fill in the downstream instance information.
+3. Click **Next**.
+
+![DM Portal InstanceConfig](/media/dm-portal-instanceconfig.png)
+
+### Configure the binlog filter
+
+This feature is used to filter the upstream binlog. You can choose the DDL or DML that needs to be filtered. The filter configured on the database is automatically inherited by tables in that database.
+
+#### Prerequisites
+
+- The upstream and downstream instance information is configured.
+- The connection is verified.
+
+> **Note:**
+>
+> - The binlog filter configuration can only be modified in the upstream instance. Once the database or table is moved to the downstream instance, the configuration cannot be modified.
+> - The binlog filter configured on the database is automatically inherited by tables in that database.
+
+#### Operation steps
+
+1. Select the databases or tables that need to be configured.
+
+2. Click the **Edit** button, and select the binlog types to be filtered.
+
+![DM Portal InstanceShow](/media/dm-portal-instanceshow.png)
+
+![DM Portal BinlogFilter 1](/media/dm-portal-binlogfilter-1.png)
+
+![DM Portal BinlogFilter 2](/media/dm-portal-binlogfilter-2.png)
+
+### Configure table routing
+
+This feature is used to perform the following operations:
+
+- Select the databases and tables that need to be synced, modify their names, and merge databases and tables
```

Comment:
Please confirm whether it's `schemas` or `databases` here. @WangXiangUSTC PTAL

Reply:
@lilin90 I suggest using `databases` to make the wording consistent with the images.

Reply:
Agree.

Reply:
OK. I'll review this document later to make sure the wording consistent 😃